v21.1/backup.md

```diff
@@ -122,7 +117,7 @@ We recommend always starting backups with a specific [timestamp](timestamp.html)
 
 This improves performance by decreasing the likelihood that the `BACKUP` will be [retried because it contends with other statements/transactions](transactions.html#transaction-retries). However, because `AS OF SYSTEM TIME` returns historical data, your reads might be stale. Taking backups with `AS OF SYSTEM TIME '-10s'` is a good best practice to reduce the number of still-running transactions you may encounter, since the backup will take priority and will force still-running transactions to restart after the backup is finished.
 
```

Comment:
~What has changed in this line? I can't tell.  They look the same but maybe I'm missing it.